_security-analytics/usage/detectors.md

```diff
@@ -35,6 +35,16 @@ To edit a detector, begin by selecting the link to the detector in the Detector
 After you select the **Alert triggers** tab, you also have the option to add additional alerts for the detector by selecting **Add another alert condition** at the bottom of the page.
 {: .tip }
 
+### Threat intelligence feeds
+
+A threat intelligence feed is a real-time, continuous data stream that gathers information related to risks or threats. The critical information in the tactical threat intelligence feed is called an “indicator of compromise” (IoC). 
```

Comment:
I'm not a fan of the last sentence here (or my rewrite), as it's a little ambiguous. Can we be more specific about what we mean by "critical information"? In other words, what information exactly is called an IoC (what makes it critical)?